tools/sotn-assets/build.go

```diff
@@ -373,7 +373,19 @@ func buildEntityLayouts(fileName string, outputDir string) error {
 			}
 		} else {
 			less = func(i, j int) bool {
-				return toSort[i].Y <= toSort[j].Y
+				if (toSort[i].Y < toSort[j].Y) {
```

Comment:
Does the result change if you remove the `>` cases? I think these are equivalent to the `<` cases

Reply:
We have to check `a < b`, `a > b` and `a == b`. There's probably a better way to write this in Go tho.

Reply:
Maybe something like:

```go
if (toSort[i].Y == toSort[j].Y) {
  if (toSort[i].SpawnID == toSort[j].SpawnID) {
    return int8(toSort[i].Slot) < int8(toSort[j].Slot)
  }
  return toSort[i].SpawnID < toSort[j].SpawnID
}
return toSort[i].Y < toSort[j].Y
```

Reply:
I checked and the `>` and `<` cases aren't equivalent when `i == j`. I think since this is modifying @Xeeynamo 's PR we should get his approval